src/ssort/_dependencies.py

```diff
@@ -44,6 +49,13 @@ def module_statements_graph(statements, *, on_unresolved, on_wildcard_import):
                 resolved[requirement] = None
                 continue
 
+            if (
+                filename.endswith("__init__.py")
```

Comment:
An unlikely edge case, but we probably want to handle filenames such as `something__init__.py`.